src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -58,12 +58,12 @@
     <XunitArguments>$(TargetFileName) $(XunitOptions)</XunitArguments>
 
     <TestProgram Condition="'$(TestHost)'!=''">$(TestHost)</TestProgram>
-    <TestArguments Condition="'$(TestHost)'!=''">$(XunitExecutable) $(XunitArguments)</TestArguments>
+    <TestArguments Condition="'$(TestHost)'!=''">$(XunitExecutable) $(XunitArguments) {XunitTraitOptions}</TestArguments>
 
     <TestProgram Condition="'$(TestHost)'==''">$(XunitExecutable)</TestProgram>
-    <TestArguments Condition="'$(TestHost)'==''">$(XunitArguments)</TestArguments>
+    <TestArguments Condition="'$(TestHost)'==''">$(XunitArguments) {XunitTraitOptions}</TestArguments>
```

Comment:
It is not run through that target and I don't believe we can make it so with the simple mechanism we're using instead of test explorer integration, etc.

It was broken, but for other reasons. A big part of the last update was to get the debugging fixed. I really want to make sure we keep it that way. It's a huge pain when it breaks. I think as long as you change TestCommandLine in a way that will leave StartArguments without the placeholder, everything will work as before.

See tests.targets before 4bcb9b1, it took explicit care to make the VS start arguments not have the placeholder. Basically, I think we want to revert the placeholder to be done as before.